plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
+        note="Whether invisibility applies to all channels. (Serves as default for channel's state when invisiblity is channel based)">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Clarification in the settings UI about global invisibility.

This change makes it clearer to the user how the global invisibility setting interacts with channel-specific settings, which improves usability.

```suggestion
        note="Indicates if invisibility is enabled across all channels. If channel-based invisibility is selected, this acts as the default setting for each channel's visibility state.">
```